nested-spire/scripts/set-env.sh

```diff
@@ -0,0 +1,116 @@
+#!/bin/bash
+
+set -e
+
+DIR="$( cd "$( dirname "${BASH_SOURCE[0]}" )" && pwd )"
+
+norm=$(tput sgr0) || true
+green=$(tput setaf 2) || true
+red=$(tput setaf 1) || true
+bold=$(tput bold) || true
+
+
+timestamp() {
```

Comment:
there is a lot of common functions can you add a common and just import on each script?